### ISSUE TITLE
Recommended teams are not shown on multiplayer setup screen

### ISSUE THREAD

**Issue by stonerl** [widelands/widelands#3928]
Please fill out the relevant sections below and delete the rest.

**Describe the bug**
When a map has team recommendations they are shown on the multiplayer setup screen between the win conditions and the start game button. These er missing.

**To Reproduce**
1. Set up a multiplayer game.
2. Choose the map Calvison or any map with team recommendations.

**Expected behavior**
Show the recommendations again

**Version:**
 - OS: macOS
 - Widelands Version: https://github.com/widelands/widelands/pull/3857/commits/15b8420ef06235ea5b787ad01bf2d9fe47cbfdd5

**Comment by jmoerschbach** [comment]
will also be fixed with #3847

**Comment by frankystone** [comment]
In debug builds it crashes with:
~~~~
src/ui_basic/panel.cc:313: virtual void UI::Panel::set_desired_size(int, int): Zusicherung »w >= 0« nicht erfüllt.
Abgebrochen (Speicherabzug geschrieben)
~~~~
